packages/bot-web-ui/src/stores/load-modal-store.js

```diff
@@ -0,0 +1,248 @@
+import { observable, action } from 'mobx';
+import { load, config, save_types, getSavedWorkspaces, removeExistingWorkspace } from '@deriv/bot-skeleton';
+
+export default class LoadModalStore {
+    @observable is_load_modal_open = false;
+    @observable active_index = 0;
+    @observable recent_files = [];
+    @observable selected_file_id = '';
+    @observable is_explanation_expand = false;
+    @observable loaded_local_file = null;
+    @observable is_open_button_loading = false;
+    recent_workspace;
+    local_workspace;
+    drop_zone;
+
+    constructor(root_store) {
+        this.root_store = root_store;
+    }
+
+    @action.bound
+    toggleLoadModal() {
+        this.is_load_modal_open = !this.is_load_modal_open;
+
+        if (this.is_load_modal_open) {
+            this.recent_files = getSavedWorkspaces() || [];
```

Comment:
`recent_files` could be a `computed` value.

Reply:
I need this to be a variable to be use in other function. (e.g line 85)